server/kusk/client.go

```diff
@@ -222,3 +224,7 @@ func (k *kuskClient) ListNamespaces() (*corev1.NamespaceList, error) {
 	}
 	return list, nil
 }
+
+func (k *kuskClient) GetK8sClient() client.Client {
```

Comment:
Small one https://go.dev/doc/effective_go#Getters

```suggestion
func (k *kuskClient) K8sClient() client.Client {
```

Reply:
Totally forgot about this.

Thanks man